Client/Application/AppDelegate.swift

```diff
@@ -62,28 +62,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     private func setUserAgent() {
         let webView = UIWebView()
```

Comment:
We should probably move this `webView` initialization into the if block below. If https://github.com/mozilla/firefox-ios/pull/492#issuecomment-105705557 is any indication, this could be the most expensive part of this method!